src/main/kotlin/com/cognifide/aem/apmt/Group.kt

```diff
@@ -0,0 +1,6 @@
+package com.cognifide.aem.apmt
```

Comment:
Let's clean up the packages, here and in the files below, skipping the `aem` bit